Modules/Calendar/Calendar/DatePicker/DatePickerAction.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import Models
+import Utils
+
+public enum DatePickerAction: Equatable {
+    case daySelected(Date)
+    case weekStripeSwiped(SwipeDirection)
+    case calendarSwiped(SwipeDirection)
+}
+
+extension DatePickerAction: CustomDebugStringConvertible {
+
+    public var debugDescription: String {
+        switch self {
+        case let .daySelected(date):
+            return "DaySelected: \(date)"
+        case let .weekStripeSwiped(direction):
+            return "WeekStripeSwiped: \(direction == .left ? "left" : "right")"
+        case let .calendarSwiped(direction):
+            return "CalendarSwiped: \(direction == .left ? "left" : "right")"
```

Comment:
You don't need this string interpolation, Swift is smart 😄 

```suggestion
        case let .weekStripeSwiped(direction):
            return "WeekStripeSwiped: \(direction)"
        case let .calendarSwiped(direction):
            return "CalendarSwiped: \(direction)"
```